phpcs.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruleset name="Zend Framework Coding Standard">
+    <rule ref="./vendor/zendframework/zend-coding-standard/ruleset.xml"/>
+
+    <rule ref="ZendCodingStandard.Commenting.FunctionComment.TypeHintMissing">
+        <severity>0</severity>
+    </rule>
+
+    <rule ref="ZendCodingStandard.NamingConventions.ValidVariableName.NotCamelCaps">
+        <exclude-pattern>src/Scanner/AnnotationScanner.php</exclude-pattern>
+        <exclude-pattern>src/Scanner/ClassScanner.php</exclude-pattern>
+        <exclude-pattern>src/Scanner/DocBlockScanner.php</exclude-pattern>
+        <exclude-pattern>src/Scanner/MethodScanner.php</exclude-pattern>
+        <exclude-pattern>src/Scanner/TokenArrayScanner.php</exclude-pattern>
```

Comment:
In these files we have variables like: `$MACRO_STREAM_ADVANCE_CHAR`. Probably we can rename all these variables there and then remove this exception here. Or is there any other reason why we have uppercase, underscore separated variable names there?